### ISSUE TITLE
Provide a Cask for Homebrew to install Octant as a Mac application

### ISSUE THREAD

**Issue by EronWright** [vmware-tanzu/octant#581]
Would it be possible to install Octant such that it were available as a normal Mac application?   I believe that the cask feature of Homebrew makes that possible.    Thanks!

**Comment by bryanl** [comment]
Yes. This is possible and we tackle it in our ongoing work to make Octant a full application.

**Comment by wwitzel3** [comment]
Let's get this done for 0.20 for the `octant-desktop` formula

**Comment by xtreme-vikram-yadav** [comment]
picking it up.
